src/views/arrange/index.js

```diff
@@ -62,20 +56,81 @@ export class Arrange extends Component {
           ...response.data.arrangement,
           unsnapshot_items: items
         }
-        this.setState(stateVal)
+        this.props.setRealData(stateVal)
         return Promise.resolve();
       })
       .catch(err => {
         return Promise.reject(err)
       })
+    // For test without API call  

```

Comment:
I think we can get rid of this now. Since we can start to generate our own data now.